tests/suites/cloud/suite.js

```diff
@@ -113,12 +112,19 @@ module.exports = {
 
     // create a balena application
     this.log("Creating application in cloud...");
-    await this.context.get().cloud.balena.models.application.create({
-      name: this.suite.options.id,
+    const app = await this.context.get().cloud.balena.models.application.create({
+      name: this.context.get().balena.name,
       deviceType: this.suite.deviceType.slug,
       organization: this.context.get().balena.organization,
     });
 
+    this.suite.context.set({
+      balena: {
+        name: app.app_name,
```

Comment:
Wouldn't this be same as this.suite.options.id?

Reply:
Yes it should, but if for some crazy reason the API sanitized our string I would prefer we use the name the SDK returned, just in case.

Reply:
Same with the slug, I want to be sure we use the slug as returned by the SDK, not just assume <org>/<name>.